### DISCUSSION TITLE
How to make the app more popular?

### DISCUSSION THREAD

**Discussion by johannesjo** [super-productivity/super-productivity#3586] [General]
Having a lot of users is important for this app to keep getting better and to survive in the long run. I would be grateful for any suggestion on how to make the app more popular. Which bloggers and youtubers might I reach out to? What reddit channels might be interesting? What other possibly unexplored avenues are there?

**Comment by dictionarymouse** [comment]
Videos on how to **easily** 😉 setup a Webdav server to privately sync your files. 
If it cannot be outright easy, make it as clear as possible, walkthrough possible steps, install this, copy/paste this into that app... etc.
I've seen lots of feature requests on different productivity apps for local, WiFi, privacy-preserving sync methods now that all the big companies are serving our data to their clouds. 

There are youtubers, rumble, odysee, twitter-x, etc. channels dedicated to data privacy and security, maybe they could be reached out to? 

Methinks that's an untapped market, but setup needs to be a little easier or clearer though. (Assuming it is possible/insofar as it is possible; I'm still trying to figure it out for apple devices.)

**Reply by bytrangle** [reply]
Any example of productivity apps with privacy-preserving data synchronisation?

**Comment by dictionarymouse** [comment]
Another possibility, update the Productivity blog on the website with productivity advice and use-case scenarios showing off Super-Productivity's features, so the site itself is a place of interest and a source of information. The site could then do a little of its own marketing. 

A picture is worth a thousand words... some thumbnails on the productivity blog might help make it more interesting.

**Comment by Jaeoc** [comment]
There is a large productivity community surrounding the app Obsidian (as well as a subreddit), with many content creators (e.g., Nicole van der Hoeven). I image there might be an interest in that community for the Superproductivity app as well.

**Reply by AethelVeritas** [reply]
There are like two Todoist integrations for use with Obsidian, as well as one for TickTick..maybe a superproductivity integration? Though honestly I think that would be a lot of work for little reward.

**Reply by bepolymathe** [reply]
Personally, I think that such integration would open the public from IT workers to academics.

**Reply by scottaltham-payroc** [reply]
Yeah this 100%. I only discovered this app thanks to Google Gemini suggesting me an open source app were I can sync my own files. That whole ownership of your data is paramount to Obsidian users I think.

**Comment by AethelVeritas** [comment]
I guess this is might be something you've heard a lot before, but maybe more modern UI?

**Reply by bytrangle** [reply]
That will probably require an actual designer to make sure the UI is coherent throughout.

**Reply by AethelVeritas** [reply]
Yeah I guess that's the issue...

**Reply by johannesjo** [reply]
Are there specific parts that strike you as not modern or is it the general look and feel?

**Reply by AethelVeritas** [reply]
Maybe not modern is not the right phrasing... I'd say it's not as modern as other more popular todo apps. Like I said on the subreddit: checkboxes on the right, dashboard blending in with the task, and I'd say having the tasks squashed together with so little spacing is kind of overwhelming...harder to identify each individual task separately. Also this might be just me, but I really do not like how rounded the corners are in the schedule view for the time-blocks... 
I wish I could contribute to this... I'll try and learn some css and maybe I can eventually make a custom theme, or at least part of one... that would be a fun project.

**Reply by bytrangle** [reply]
You can help spreading the word, or give your input some issues. There are plenty issues that can only be moved forward with community feedback, and you don't need to have technical knowledge because this app is used by both tech and non-tech people.

**Reply by scottaltham-payroc** [reply]
I think it's quite modern to be fair. A little more padding between UI elements would be nice, but otherwise, I've got mine looking rather lovely!

**Comment by rthorntn** [comment]
Not sure if you use reddit... /r/selfhosted would be worth posting to:

https://www.reddit.com/r/selfhosted/comments/1jkxdmo/app_to_organise_all_the_things_i_have_to_do/?sort=new

30K views and it brought me here.

/r/productivity as well.

**Comment by jpgaubier** [comment]
I just discovered Super Productivity through a tech news article and wow, it's fantastic. I think more people should know about it, for sure! One addition that could boost and broaden its appeal is more integrations with general team project management software, such as Asana. This would be a good complement as an individual's tasks on a team (in Asana)  could be managed personally (in Super). Skedpal takes this approach and is a good example, I think.

**Reply by bytrangle** [reply]
Skedpal is pretty cool but it is a **paid** app with a full-time staff. It is not comparable.

**Reply by jpgaubier** [reply]
Yes, agreed. They are different tools for different purposes. Both are excellent.

**Comment by toddz262** [comment]
Word of mouth is powerful, and I think a key to gaining users—who would then sing the praises of SP—would be the ability to easily import tasks from an existing system. SP has a great feature set and could probably replace commercial tools for many people, but when your whole life is loaded into one product, moving to another _has_ to be made easy.

In my case, SP looks like a great alternative to ToDoist, but I'm faced with manually re-entering a couple hundred tasks.

Some products can import the raw export file from leading competitors. Others provide a sample CSV import file and it's up to the user to massage their exported data into that format. Either method would make it easier for more users to come on board.

**Comment by OLLI-S** [comment]
I am a usability manager and an accessibility manager.
This also includes UX (User eXperience).

The layout of Super Productivity really could be improved.
@ModernGalahad mentioned "checkboxes on the right" and this is a good example.
Users expect in a ToDo manager that the checkbox to complete the task is on the left side (because this is the default layout for a checkbox).
Also I was very confused, when there was no checkbox.

You also have accessibility (abbreviated: A11Y) issues and some violations regarding thre [WCAG 2.2 AA](https://www.w3.org/TR/WCAG22/)
Here I can help you and send send you the issues.
@johannesjo Should I open a new issue here, where I list the examples and constantly add new findings? I would creatre one issue for each requirement, e.g. "Color Contrast" or "Visible Focus".

To make Super Productivity more popular, you can also write on your website, that it is exttremely helpful for people sufffering on ADHD (in German: ADHS), because they have massive problems staying focused.
And here Super Productivity is definitely helping.
You should also mention "neuro diversity", because this covers more than just ADHD, but for some neuro diverse people some animations might be disturbant.
The goal is: when people search for "ADHD task manager" or "ADHD todo list", then Super Productivity should be top ranked.

**Reply by johannesjo** [reply]
Thank you very much for your input! Yes, I would appreciate it a lot if you would create an issue for the things you found.

**Reply by OLLI-S** [reply]
@johannesjo I want to show you the layout of a component set, that we are using at work (here I am unsure about the license, so I will not post it here).
Can you please contact me directly (I sent you an email some days ago with my personal contact data).

**Reply by johannesjo** [reply]
@OLLI-S thank you very much! Just found your email in my spam folder :) I will reach out to you as soon as I find the time (next week is a bit busy for me as I will start a new freelance job).

**Reply by OLLI-S** [reply]
Next week I am on vacation, so I will be home the whole week (perfect time for such a meeting).
But if this does not match your calendar, then we find a different time slot.

**Reply by jpgaubier** [reply]
> Users expect in a ToDo manager that the checkbox to complete the task is on the left side (because this is the default layout for a checkbox).
> Also I was very confused, when there was no checkbox.

I take a different view on this. I think Super Productivity's current UI makes sense because it is not really a to-do list; it is a task manager with a task completion function. I move tasks around all the time and having the "handle" on the left allows one to do this. A checkbox would actually get in the way of this task-movement function. By definition, a checkbox is used only once whereas task dragging (in my case) is often done multiple times, so to me it makes sense for the completion button to be subordinate.

**Reply by OLLI-S** [reply]
> @OLLI-S thank you very much! Just found your email in my spam folder :)

Maybe you add me to the "Friends List", so you don't miss my emails?

**Comment by SimmelGeorg** [comment]
Integration and collaboration with Joplin Note App. They are also open-source and they have a nice community around the app like here. Their main focus is note taking therefore elaborated task/to-do manager functionalities are missing (for an understandable reason). I believe SuperProductivityApp could cover that gap!

**Comment by OLLI-S** [comment]
I played a bit with Obsidian but also had a look at Joplin.
Joplin is simpler to set up and easier to use (not such a complexity monster).
I would really love to SP integrated in Joplin, especially when SP stores the data in Joplin (so the data is automatically synced with the Joplin notes).

**Reply by bepolymathe** [reply]
I think we should adopt an application-agnostic approach. Being able to create/synchronize SP notes with a local folder containing markdown files will allow compatibility with dozens of such applications (Obsidian, Joplin, Zettlr, Trillium, etc.).

**Reply by jpgaubier** [reply]
I would add Logseq to the list, too. Also uses Markdown.

**Comment by efb2a** [comment]
- Lean Heavily into Single User App Branding For Diffrentiation, There Are a Lot of Task Management Apps, Unless We Want to Bleed Userbase to Latest OSS Notion Clones , We Should Lean Heavily into Branding This App as Personal | Single User Task Management Tool. This Won't Require Any Hard Work, Merely Rewriting App Introductions on Github,Website . I Personally Came Across This App Via AI Reccomendations, Leaning into Single User / Personal Use Branding May Help With SEO and Discovery. 
- If Possible, We Can make a Video Tutorial and Shorts Series Generated By Ai Interacting With Docs and Share That on Social Media . There Are a Few Free Resources Like Gemini / Deepseek that can Help with that. Video Tutorials can Both Serve as way to Acquire New Users and Introduce Existing Users to Full Featureset .

**Comment by jakeingit** [comment]
I'm a marketing and ops guy -- I'm marketing this by literally getting clients and giving them super productivity, and coaching them through as they use it to grow their business. I'd rather they pay for coaching + support, than pay an app like Clickup that they pay $500/mo for a few seats. 

That value is in the transformation to their bottom line -- not a new feature or habit they need to be on top of.

You need to have financial incentives -- for affiliates to even want to push this.

Let me know if you want to discuss or brainstorm stuff.

**Reply by johannesjo** [reply]
I would love to chat about this with a professional. I really appreciate the offer! Can you maybe reach out via contact@super-productivity.com so we can set something up?

**Reply by jpgaubier** [reply]
> That value is in the transformation to their bottom line -- not a new feature or habit they need to be on top of.
> 
> You need to have financial incentives -- for affiliates to even want to push this.

That is one model but it isn't the only one. See Futo.org or Opencollective.com as examples of others that FOSS projects have used successfully (e.g., Immich, Logseq). 

As for affiliate marketing, I would say it's often obvious, not in a good way.

**Comment by jakeingit** [comment]
But TBH, integrate this with Obsidian -- and it'll be able to integrate with local agentic AI agents to do tasks.

There is a huge push in the business space to use n8n automations, and local AI to automate a business.

If AI can know how to handle tasks, in a way that humans can see, and feed it ideas and learning and context, then it makes it all very spicy and profitable to keep running.

**Reply by LeonelRFF** [reply]
As for it becoming more popular, today the presence of obsidian is still noticeable, I also think it's a good idea to integrate it in some way with this tool

**Comment by jpgaubier** [comment]
> But TBH, integrate this with Obsidian -- and it'll be able to integrate with local agentic AI agents to do tasks.
> 
> There is a huge push in the business space to use n8n automations, and local AI to automate a business.
> 
> If AI can know how to handle tasks, in a way that humans can see, and feed it ideas and learning and context, then it makes it all very spicy and profitable to keep running.

As a business owner, I get solicitations for a new AI integration every single day, from  companies I am not too sure will be around next month. Yes, there is a huge push because AI is awash in VC. But the AI is mostly oversold. I would not put too many resources into integrations with fleeting AI tools that overpromise and underdeliver, especially if they are not open source (which is really the only thing that assures their dependability and potential longevity).

**Reply by jakeingit** [reply]
I use this for myself, and its making me money. But yeah -- before AI you and I were cold called by marketing kids promising leads and wealth.

I use it to do a lot of creative tasks and manual work that I don't need to do.

What I do is feed it 1000s of pages of notes and documentation, all the stuff I wrote instructions on and it solves problems for me and my team.

**Comment by MrDaisyBates** [comment]
Many great things in life come with some effort behind them.

I think the most important parts are a great product and patience. 

However, seeing is believing. I think ample quality youtube videos (a few long ones is fine, but shorter ones on features, as well as how to accomplish certain methods using your software).

I'm not super savvy with the "productivity" world, but IMO, if you illustrated really clearly how to do a "Getting Things Done (GTD)" tutorial, (and then ones for other popular workflows or whatever), that already is going to help a lot.

Word of mouth will take care of things in the long run, but again, patience is the key there. A great product will eventually rise to the top. But if that's still too slow for your liking, I'm a firm believer in "give value, get value." You've done a lot, but if you want faster for whatever reason, quality youtube tutorials would be a great start. 

On that note, it's really important to understand that if you RETAIN most of the people you capture, your rates of finding new people can be low. I'd focus on ensuring that you're really pleased with the performance (bugs, stability, etc) as well as ensuring there is good documentation. 

Most people who shop for software kinda feel out what's there in the space, and if the first days are frustrating, you're a bit less likely to keep them. I think the tutorial is a HUGE step in the right direction, but banging documentation and tutorials will significantly boost retention, I feel. 

Ultimately, I think you have a fine product and I can tell you've put some effort into it. Consistency and patience will do wonders in this area I feel. 

I've already told some people about the software today (just today started messing with it, too). It will be part of my workflow going forward because it seems to check my boxes, but I'd love to see more documentation personally. As an example, I use Tangent (markdown note taking software. Quite niche). And I originally downloaded it like 4 to 6 months ago. But it sat there unused for three months because I was overwhelmed with how good it probably was...but I didn't find the documentation for whatever reason. It could have been dead forever with me. But then I DID find the documentation on a second revisit and now it's my daily driver for notes. 

My point is, the software was great, and a great fit for me, and it was only by chance that I even found it...but it almost ended up being a flop despite being almost perfect for me. So, even getting my attention to download it and it being a perfect fit wasn't enough. He also put a website up with a demo of the note taking style he designed the software around...which helped me "see" what it could do. The documentation even might not have been enough.

Anyways, I'm just trying to help out because I like what you're doing and I'm just speaking from what I've seen. A picture says a thousand words. I think you've done well with your site and stuff. I'd love to see a Discord personally. And it might be smart on your part to make videos teaching people sound principles around productivity or something (while using your software). Focus on giving value that can be used with ANY software basically, and that video is more likely to trend and get attention. When you create that value and goodwill for others, it comes back to you, too. If it just feels like a plug for your software, it probably won't do as well. Make separate playlists on your youtube for each of these categories, too. Like one for the quick and dirty tutorials, one for more advanced, longer tutorials, and then one playlist of like productivity theory or something that purely focuses on universal principles that apply to all scenarios/software/etc. It can be the free value that you offer others out of your goodwill and will get you in front of more people.

That's all I've got to say. I appreciate what you're trying to do here. I need to go figure out your software now. Hahaha. That's how I found this thread. I was trying to find help here in your discussions.

**Reply by MrDaisyBates** [reply]
I did want to express some other thoughts around these same points and why they are maybe more important than one might think at first glance.

For example, I'm willing to dig in and figure it out. I have some specific needs right now and I think they are built into the software, and I'm an open source kinda guy, so, I'm an easy sell.

However, I've got some very hard working and professional people that I'm unwilling to really promote your software to just yet. Why? Because I can't answer their questions about how to use it, and they can't really see in a quick manner maybe how it fits into their grind better than what they already use (even if it IS better). It makes it tough to suggest. Whereas Tangent, my note taking software I use, despite it being super tiny, it was easy for me to promote because I was able to easily show why it was so powerful for me, as well as point them towards documentation. 

My point is, if others feel the way I do, you're going to have a hard time getting solid word of mouth because they're going to hesitate. 

Another negative consequence that I'm certain you experience is that all the users you have coming on see your software, get excited, and try to use it...and intuitively, maybe they're looking in a certain place for a certain detail...and it's not there. And because of it, they either scrap it or they start making requests for new features...

However, if they fully understood all the features that already existed, they might realize that they don't need the feature (maybe it's redundant) or maybe you've actually come up with a more intuitive way to do things, making their feature request a bit more irrelevant. 

I'd argue that great tutorials and documentation and some general principles for success (like the generalized how to I described above) might help you better train people how the software is kinda meant to function. This will help keep you from getting overwhelmed with bloat and help reduce you losing sight of what you set out for. 

Anyways, I just wanted to add these points since they seem relevant and should offer you some insight. I know you know the software so well because you're intimate with it, but as a new comer, the experience is very different, of course! 

Again, not to detract from what great stuff you've already done. I'm quite impressed. You asked for feedback in this area, so I'm simply offering it as a way of thanking you for what you've done.

**Reply by Sergih28** [reply]
> However, I've got some very hard working and professional people that I'm unwilling to really promote your software to just yet. Why? Because I can't answer their questions about how to use it, and they can't really see in a quick manner maybe how it fits into their grind better than what they already use (even if it IS better). It makes it tough to suggest.

I relate to this.
I want to improve my knowledge and experience using superproductivity before recommending it to non-advanced people.

Also, I find it weird that, even though I usually don't look for specific open source software, I've done a lot of research the past years trying and checking several todo apps, and as far as I can remember, superproductivity never came up, it was a workmate that recommended it to me recently.
Which is weird, because I've read lots of articles and watched several videos.

I might've been just unlucky or something, idk, but logseq and other popular open source and non open source tools came up when I was looking for those specific types of tools.
For example obsidian, has several videos from some popular youtubers within the note taking world, and it's hard to find something about superproductivity, unless you are already know about its existence. And even with that, there aren't many videos like I've seen with obsidian that get your attention and help you get you started.

I'm no expert in marketing or anything, but might be worth to look into what other tools like logseq/obsidian/notion have done to become popular, because superproductivity is really good.

**Reply by pgpbj** [reply]
> Also, I find it weird that, even though I usually don't look for specific open source software, I've done a lot of research the past years trying and checking several todo apps, and as far as I can remember, superproductivity never came up, it was a workmate that recommended it to me recently.
Which is weird, because I've read lots of articles and watched several videos.

This is my experience as well.  I am constantly searching for well-made opensource productivity apps, especially ones that are privacy-centric and cross-platform and I have only learned of SP **yesterday** (I stumbled upon a FLOSS weekly podcast interview with Johannes) even though it appears to have been in existence for 10 years!

Recently I have seen that a number of privacy-related apps and sites are banding together and essentially promoting each other.  I would have learned of SP sooner if it were able to break into this circle.  I am thinking of Notesnook.com, Ente.io, Tuta.com, Addy.io, etc.  Also PrivacyGuides.org has a list of recommended software tools -- there is currently no category that maps directly to SP, but maybe they could add one!?

**Reply by johannesjo** [reply]
Thank you @pgpbj ! That sounds like a great suggestion! I look into it!

**Comment by darius-r** [comment]
I found this app because of Reddit recommendations and I'm honestly confused how I overlooked it for so long. I sorely wish I had found it much sooner.

I'm arriving here mainly from [LogSeq](https://logseq.com/) which I gave up on adapting myself to. I lost trust in the development cycle and splitting between MD and database versions. 

What you've built, even with its immediate limitations, is a better personal project/task management solution than everything else I've tried. 

How you've approached developing the application and communicating with users and contributers alike, really tied the knot for me.  Its incredibly professional, productive, honest, and patient, even when the tone of the people writing to you isn't. Its the type of "be the change you want to see in the world" we need more of, and clearly motivates people to want to contribute to the collective improvement of it. 

My suggestion: Find a way to promote SP to people using Proton.me and try to poach users from Standard Notes. https://standardnotes.com/

With everything that's happening in the world, I honestly think the push for 'local first' is going to become exponentially more significant than previous years. 

Happy 15th anniversary by the way! Its admirable what you've done.

**Reply by johannesjo** [reply]
Thank you so much for the kind words! I really appreciate it!
